packages/server/src/api/controllers/pools.ts

```diff
@@ -134,10 +134,40 @@ async function getHistoricalHourlyData(req: Request<PoolPath, unknown, unknown,
 }
 
 const route = Router();
-route.get('/:network/ethPrice', networkValidator, catchAsyncRoute(getEthPrice));
-route.get('/:network/pools', getTopPoolsValidator, catchAsyncRoute(getTopPools));
-route.get('/:network/pools/:poolId', poolIdValidator, catchAsyncRoute(getPoolOverview));
-route.get('/:network/pools/:poolId/historical/daily', getHistoricalDataValidator, catchAsyncRoute(getHistoricalDailyData));
-route.get('/:network/pools/:poolId/historical/hourly', getHistoricalDataValidator, catchAsyncRoute(getHistoricalHourlyData));
+const cacheConfig = { public: true, mustRevalidate: true };
+const poolConfig = { maxAge: 10, sMaxAge: 60, ...cacheConfig };
```

Comment:
for getEthPrice, /pools, and pool/:id
clients must revalidate every 10s
cdn revalidates every 60 (due to cache warmer)
cache warmer & memoize every 5 min